tests/api.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require('supertest');
+const db = require('../server/db.js');
+const app = require('../server/app.js');
+
+afterAll(() => db.sequelize.close());
+
+describe('Api should get data from database', () => {
+  test('It should respond with an array of user objects', () => {
+    return request(app).get('/api/users')
+      .then(response => {
+        expect(response.body.length).toBe(50);
```

Comment:
Hard coding to a number of records in a database is not recommended unless you're clearing and reseeding the db before each test.